synapse/common.py

```diff
@@ -40,6 +41,9 @@ def addpref(pref, info):
     '''
     return {'%s:%s' % (pref, k): v for (k, v) in info.items()}
 
+# XXX Tufo-type for type annotation
+tufotype = ('', {})
```

Comment:
I was experimenting with some type hinting; its way easier to do this if we had access to the actual typing module. This is going to be removed.